### PR TITLE
rm unittests eager guard test part8 gaussian2gumnel

### PR DESCRIPTION
**Pull request by yjjiang11**
### PR types

Others
### PR changes

Others
### Describe

rm unittests eager guard test part8 gaussian2gumnel

### CODE REVIEW

**Review by wanghuancoder — DISMISSED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/test_group_norm_op.py [hunk 1]
```diff
@@ -307,20 +306,15 @@ def test_dygraph_api(self):
...
-                ret2.backward()
-                self.assertEqual(
-                    (
-                        tensor_1.grad.numpy() == tensor_eager_1.grad.numpy()
-                    ).all(),
-                    True,
-                )
+            tensor_eager_1 = fluid.dygraph.to_variable(input)
+            tensor_eager_1.stop_gradient = False
+            groupNorm_eager = fluid.dygraph.nn.GroupNorm(channels=32, groups=4)
```
Comment (wanghuancoder) [inline python/paddle/fluid/tests/unittests/test_group_norm_op.py]: paddle.nn.GroupNorm
改成这个试试

**Comment by yjjiang11** [comment]
6fail

**Review by veyron95 — APPROVED** [review]
LGTM

**Pull request merged**